pandas/core/generic.py

```diff
@@ -3580,6 +3618,44 @@ def tail(self, n=5):
         -------
         obj_tail : type of caller
             The last n rows of the caller object.
+
+        See Also
+        --------
+        pandas.DataFrame.head
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'animal':['falcon', 'parrot', 'lion',
```

Comment:
One last typo, this dataframe should also include alligator, bee... it's inconsistent with the results now.

Reply:
fixed :+1: